src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -717,8 +705,8 @@ bool Frame::write_aov_images(const char* file_path) const
         const string aov_file_path = (directory / aov_file_name).string();
 
         // Write AOV image.
-        if (!write_image(aov_file_path.c_str(), aov->get_image(), aov))
-            success = false;
```

Comment:
Why the change of logic here?

Reply:
You are asking about `success` variable, right? Sorry I misunderstood. It's because of the for loop. I thought breaking the loop and returning false on first failing aov.

Reply:
Yes, the `success` variable. I prefer that we don't change the logic, if possible.